polaris/ocean/tests/baroclinic_channel/baroclinic_channel.cfg

```diff
@@ -0,0 +1,77 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 20
+
+# Depth of the bottom of the ocean
+bottom_depth = 1000.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
+
+# config options for baroclinic channel testcases
+[baroclinic_channel]
+
+# the number of mesh cells in the x direction
+nx = <<<set in code>>>
+
+# the number of mesh cells in the y direction
+ny = <<<set in code>>>
+
+# the distance between adjacent cell centers
+dc = <<<set in code>>>
+
+# the number of cells per core to aim for
+goal_cells_per_core = 200
```

Comment:
Do you think it makes sense to have this and `max_cells_per_core` be defined for each test case rather than in a more general section, perhaps `parallel` in `default.cfg`?

Reply:
That's a very good suggestion.  I think it's somewhat component specific so probably belongs in `ocean.cfg`.